specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-03-01-preview/examples/GetDiagnostics.json

```diff
@@ -0,0 +1,38 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "location": "eastus"
+  },
+  "responses": {
+    "200": {
+      "headers": {
+        "x-ms-request-id": "27b7c568-16ec-46f3-bcf1-5bea3f2529b1"
+      },
+      "body": {
+        "value": [
+          {
+            "id": "diskInspection",
+            "name": "Disk Inspection",
+            "type": "Microsoft.Compute/locations/diagnostics",
+            "properties": {
+              "supportedResourceTypes": [
+                "Microsoft.Compute/virtualMachines"
+              ]
+            }
+          },
+          {
+            "id": "spotPlacementRecommender",
+            "name": "Spot Placement Recommender",
+            "type": "Microsoft.Compute/locations/diagnostics",
+            "properties": {
+              "supportedResourceTypes": [
+                "Microsoft.Compute/virtualMachines"
+              ]
+            }
+          }
+        ],
+        "nextLink": null
```

Comment:
>null [](http://example.com/codeflow?start=20&length=4)

its usually discouraged to include explicit nulls in the response, as opposed to omitting them - do you need them in the example? (Are the fields x-nullable true? It seems they are not, in which case there should be no explicit 'null's in the response)

Reply:
Removed nextLink: null from example file - no example of this field being set in the current API version. Will also update our API to omit nextLink in response when value is null